Gem/Code/Source/Weapons/SceneQuery.cpp

```diff
@@ -57,7 +57,7 @@ namespace MultiplayerSample
                 IntersectResult intersectResult;
                 intersectResult.m_position = hit.m_position;
                 intersectResult.m_normal = hit.m_normal;
-                intersectResult.m_materialName = hit.m_material->GetSurfaceTypeName();
+                intersectResult.m_materialName = hit.m_physicsMaterialId.ToString<AZStd::string>();
```

Comment:
Not really your code, but if this has turned into a meaningful Id value (hopefully strong-typedef'd for type safety?), then we should probably get rid of the material name and pass through the Id directly..